__tests__/intergration.tests.js

```diff
@@ -139,13 +139,94 @@ describe("/api", () => {
   });
 });
 
+describe("/api/articles/:article_id/comments", () => {
+  test("POST: 201 accepts an object with username and body, returns the posted comment", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        body: "testing, testing, 123!",
+        username: "butter_bridge",
+      })
+      .expect(201)
+      .then((response) => {
+        expect(response.body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          article_id: 1,
```

Comment:
We should be expecting the exact values of the new comment properties, including accurate `comment_id`